recipes/enchant/all/CMakeLists.txt

```diff
@@ -0,0 +1,118 @@
+cmake_minimum_required(VERSION 3.12)
```

Comment:
An attempt was made to use upstream build system: https://github.com/friendlyanon/conan-center-index/tree/package/enchant-backup/recipes/enchant/all  
It never behaved correctly on Windows.

CMake 3.12 is required because [GenerateExportHeader](https://cmake.org/cmake/help/latest/module/GenerateExportHeader.html) added support for C targets in 3.12.